baselines/FedSMOO/pyproject.toml

```diff
@@ -0,0 +1,135 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "FedSMOO" # <----- Ensure it matches the name of your baseline directory containing all the source code
```

Comment:
```suggestion
name = "fedsmoo" # <----- Ensure it matches the name of your baseline directory containing all the source code
```

Reply:
As mentioned in my main comment, please rename your baselines directories to `fedsmoo` (lowercase)